gguf-py/gguf/constants.py

```diff
@@ -23,14 +23,24 @@ class General:
         QUANTIZATION_VERSION = "general.quantization_version"
         ALIGNMENT            = "general.alignment"
         NAME                 = "general.name"
+        BASENAME             = "general.basename"
+        FINETUNE             = "general.finetune"
         AUTHOR               = "general.author"
+        ORGANIZATION         = "general.organization"
         VERSION              = "general.version"
+        BASE_VERSION         = "general.base_version"
         URL                  = "general.url"
         DESCRIPTION          = "general.description"
         LICENSE              = "general.license"
+        LICENSE_NAME         = "general.license.name"
+        LICENSE_LINK         = "general.license.link"
         SOURCE_URL           = "general.source.url"
         SOURCE_HF_REPO       = "general.source.huggingface.repository"
         FILE_TYPE            = "general.file_type"
+        PARAMETER_SIZE_CLASS = "general.parameter_size_class"
+        TAGS                 = "general.tags"
+        LANGUAGE             = "general.language"
```

Comment:
Maybe it should be `general.languages` plural, because it's a list of strings

Reply:
The key is `language` in the HF API. Doesn't really matter to me, but matching it would simplify it.

https://huggingface.co/docs/huggingface_hub/guides/model-cards#include-evaluation-results